src/json-utils.js

```diff
@@ -50,3 +50,52 @@ function getAttributes(entity) {
   }
   return elemObj;
 }
+
+function createEntities(EntitiesJSON) {
+  const entitiesData = EntitiesJSON.data;
```

Comment:
You should call the parameter in the function `entitiesData`
and use the `createEntities` function with createEntities(EntitiesJSON.data)
so the `createEntities` is not aware of the `EntitiesJSON` structure with the data key in the json.
Later we could have other keys in the json, like `EntitiesJSON.permissions`, but this is backend implementation specific, no need for this createEntities function to be aware of it.

Reply:
see my comment https://github.com/3DStreet/3dstreet/issues/212#issuecomment-1283639723